src/vmm/src/lib.rs

```diff
@@ -46,13 +46,18 @@ use crate::device_manager::mmio::MMIODeviceManager;
 use crate::memory_snapshot::SnapshotMemory;
 #[cfg(target_arch = "x86_64")]
 use crate::persist::{MicrovmState, MicrovmStateError, VmInfo};
+use crate::rpc_interface::{ActionResult, VmmActionError, VmmData};
+use crate::vmm_config::balloon::{
```

Comment:
Same with `vmm_config`: this is something RPC/API specific. The `vmm` implementation doesn't need to know about it (depend on it).